src/vellum/workflows/nodes/displayable/tool_calling_node/utils.py

```diff
@@ -33,17 +34,20 @@ class FunctionNode(BaseNode):
 
 class ToolRouterNode(InlinePromptNode):
     max_prompt_iterations: Optional[int] = 5
+    state: ToolCallingState
```

Comment:
delete this. Instead, we parameterize the node with `class ToolRouterNode(InlinePromptNode[ToolCallingState]):`